liota/dccs/iotcc.py

```diff
@@ -146,6 +146,29 @@ def on_response(msg):
 
             return RegisteredEntity(entity_obj, self, self.reg_entity_id)
 
+    def unregister(self, entity_obj):
+        """ Unregister the objects
+        """
+        log.info("Unregistering resource with IoTCC {0}".format(entity_obj.ref_entity.name))
+
+        def on_response(msg):
+            try:
+                log.debug("Received msg: {0}".format(msg))
+                json_msg = json.loads(msg)
+                log.debug("Processed msg: {0}".format(json_msg["type"]))
+                if json_msg["type"] == "remove_resource_response":
```

Comment:
Combine this two "if" loop by using && condition.
You don`t have the else loop in case the first if loop fails.